update-sdk/src/index.ts

```diff
@@ -3,15 +3,24 @@ import * as core from '@actions/core';
 
 async function main() {
   const repository = core.getInput('repository');
+  const changeset = core.getInput('changeset') === 'true';
   const branch = core.getInput('branch');
   const npmTag = core.getInput('npm-tag');
 
   const inlinePackages = core.getInput('packages');
   const isMultiline = inlinePackages.includes('\n');
   const packages = isMultiline ? core.getMultilineInput('packages') : inlinePackages.split(',');
 
-  const bot = new ReleaseBot(repository, branch, npmTag, packages);
-  await bot.release();
+  const bot = new ReleaseBot(repository, changeset, branch, npmTag, packages);
+
+  try {
+    const { hasUpdates, branch, pr } = await bot.release();
+    core.setOutput('has-updates', hasUpdates);
+    core.setOutput('branch', branch);
+    core.setOutput('pr', pr);
```

Comment:
We need to add this outputs on the README file for documentation and also define them on the action.yml

Reply:
https://github.com/FuelLabs/github-actions/pull/28/commits/c87963da141f0e149761ddef218998d3c9bf4c2a